cms/static/sass/_build.scss

```diff
@@ -67,6 +67,7 @@
 @import 'views/group-configuration';
 @import 'views/video-upload';
 @import 'views/certificates';
+@import 'views/creative-commons';
```

Comment:
Hmm, yeah, I forgot we use "elements" currently. To mirror where I'm recommending we place that ``creative-commons`` partial in the LMS, I'd recommend moving this fella to https://github.com/edx/edx-platform/tree/sass-output-to-css-dir/cms/static/sass/elements. Sound good?